content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -0,0 +1,123 @@
+    {{< /text >}}
+
+
+1. Choose one of the following two
+**mutually exclusive** options described below.
+
+### Option 1: Install with Helm via `helm template`
+
+1. Render Istio's core components to a Kubernetes manifest called `istio.yaml`:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system 
+      --set security.enabled=false \
+      --set ingress.enabled=false \
+      --set gateways.istio-ingressgateway.enabled=false \
+      --set gateways.istio-egressgateway.enabled=false \
+      --set galley.enabled=false \
+      --set sidecarInjectorWebhook.enabled=false \
+      --set mixer.enabled=false \
+      --set prometheus.enabled=false \
+      --set global.proxy.envoyStatsd.enabled=false \
+      --set pilot.sidecar=false > $HOME/istio-minimal.yaml
+    {{< /text >}}
+
+1. Install the Pilot component via the manifest:
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    $ kubectl apply -f $HOME/istio-minimal.yaml
+    {{< /text >}}
+
+### Option 2: Install with Helm and Tiller via `helm install`
+
+This option allows Helm and
+[Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components)
+to manage the lifecycle of Istio.
+
+1. If a service account has not already been installed for Tiller, install one:
+
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/helm-service-account.yaml
+    {{< /text >}}
+
+1. Install Tiller on your cluster with the service account:
+
+    {{< text bash >}}
+    $ helm init --service-account tiller
+    {{< /text >}}
+
+1. Install Istio:
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio --name istio-minimal --namespace istio-system 
+      --set security.enabled=false \
+      --set ingress.enabled=false \
+      --set gateways.istio-ingressgateway.enabled=false \
+      --set gateways.istio-egressgateway.enabled=false \
+      --set galley.enabled=false \
+      --set sidecarInjectorWebhook.enabled=false \
+      --set mixer.enabled=false \
+      --set prometheus.enabled=false \
+      --set global.proxy.envoyStatsd.enabled=false \
+      --set pilot.sidecar=false
+    {{< /text >}}
+
+1. Ensure the `istio-pilot-*` Kubernetes pod is deployed and its container is up and running:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                     READY     STATUS    RESTARTS   AGE
+istio-pilot-58c65f74bc-2f5xn             1/1       Running   0          1m
+{{< /text >}}
+
+## Uninstall
+
+* For option 1, uninstall using `kubectl`:
+
+    {{< text bash >}}
+    $ kubectl delete -f $HOME/istio-minimal.yaml
+    {{< /text >}}
+
+* For option 2, uninstall using Helm:
+
+    {{< text bash >}}
+    $ helm delete --purge istio-minimal
+    {{< /text >}}
+
+    If your Helm version is less than 2.9.0, then you need to manually cleanup extra job resource before redeploy new version of Istio chart:
```

Comment:
this needs to be done independent of Helm versions.  Helm 2.10 with `crd-install` treats crds as an unmanaged object - meaning Helm doesn't know anything about them or how to delete them. :(

Reply:
after re-reading, I see you are speaking about the job resources.  I think this needs to be done for Helm 2.10 as well - but not certain.  Why would helm 2.10 not need this operation?

Reply:
I was mainly copying this note from our helm install doc:  https://istio.io/docs/setup/kubernetes/helm-install/#uninstall

@costinm @andraxylia @gyliu513 maybe you know why?

Reply:
I think someone may have mentioned 2.9 handles better with jobs cleanup, but I am not seeing it on the https://github.com/helm/helm/releases.

Reply:
@linsun @sdake I can see most of the jobs are creating CRDs, but with helm 2.10, we will able to use `crd-install`, and no job is required, so we will not need to delete jobs.

So here, I think maybe we need to mention helm version 2.10? As helm 2.10 introduced  `crd-install` which can enable us get rid of jobs.

Reply:
I confirm, it is because helm < 2.10 has problems with CRDs installation.

Reply:
thanks @gyliu513 @andraxylia for the clarification.